app/src/androidTest/java/com/nextcloud/talk/utils/VibrationUtilsTest.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * Nextcloud Talk application
+ *
+ * @author Mario Danic
+ * Copyright (C) 2017-2019 Mario Danic <mario@lovelyhq.com>
```

Comment:
please change the author in the copyright headers

Reply:
Oh then can i write my name?

Reply:
Done